### ISSUE TITLE
[New build - DEFAULT] 24w38a: Terrain Performance, Power Cell

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#6000]
The terrain engine performance has been improved, minimizing hiccup during new terrain patch loading.
Now moving over the terrain at high speed is much, much smoother.

The power cell have been reworked and improved. 

Improvements and community Suggestions:
Power cell: smaller size is added
Power cell: setting has been removed, it now charge / discharge automatically.
#5982 Collector: Increased range
Codex item description: text overflows are fixed 
Codex item description: icons for grid scalable and stretchable are displayed. 
Scanner: will no longer show underground patch as to reward and promote exploration with hovercraft.
Scanner: Scanner visibility improvements

Hotfixes:
#5988 Power cell base does not turn off
#5779 Power cell base does not update stats on delete 
#5779 Inconsistent power display in preview

**Comment by Asnro** [comment]
Running a new Survival game, I noticed immediately that gold seems to be *more* frequent but occurring in much smaller patches. The frequency goes against your statement of gold being rarer, but I like it a lot: it gives players a much quicker start into powered crafting and hovercraft, as opposed to spending tens of minutes on foot at the start, while the small patch size means they have to go out looking for more to advance the game further.

**Comment by Asnro** [comment]
The Crucible menu seems to mess with ore/ingot colors
![kuva](https://github.com/user-attachments/assets/54340590-8fd8-4628-8447-c29a7b2a63a3)
This seems to only affect the UI icon color, the 3D models are the correct color.

**Comment by tsunamayo** [comment]
@Asnro Thanks for the quick feedback I will have a look at the issue. 
For the ore I might need to tweak it down a little more we will see. It is highly random, so you might find several patches near your place and nothing at some other.

**Comment by COMICLER** [comment]
I was working with collectors, and i have a question about the pull range because i have the largest one at 
80 and it seems to only pull items in if they're closer than 8 meters. my question being, what does 80 represent because i assumed it was 80 meters but it seems its just 8

**Comment by tsunamayo** [comment]
@COMICLER You need a larger one for that, the bigger the larger the pull.

**Comment by BigBadKangaroo** [comment]
> I was working with collectors, and i have a question about the pull range because i have the largest one at 80 and it seems to only pull items in if they're closer than 8 meters. my question being, what does 80 represent because i assumed it was 80 meters but it seems its just 8

Yeah, I'm confused too. My 1m collector says it has a 20 meter pull range ... however, after testing the actual pull range seems to be around 6 meters, and there's no way to change this.

**Comment by tsunamayo** [comment]
@BigBadKangaroo I am looking into it!

**Comment by tsunamayo** [comment]
@BigBadKangaroo Hi the range issue is now fixed. Thanks

**Comment by tsunamayo** [comment]
@Asnro Hi, issue is also fixed! Thanks for letting me know.
